downloads/1.2.x-release-notes/1.2.x-bug-fixes-template.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: ballerina-left-nav-release-notes
+title: <RELEASE_VERSION>
+permalink: /downloads/1.2.x-release-notes/<RELEASE_VERSION>/
+active: <RELEASE_VERSION>
+redirect_from:
+    - /downloads/1.2.x-release-notes/
+---
+
+## Overview of jBallerina <RELEASE_VERSION>
+
+<!-- COMMENT OUT ONLY THE APPLICABLE ONE FROM THE BELOW -->
+
+<!-- The jBallerina <RELEASE_VERSION> patch release improves upon the <PREVIOUS_RELEASE_VERSION> release by addressing an [issue](<GITHUB_LINK_OF_ISSUE>). -->
+
+<!-- The jBallerina <RELEASE_VERSION> patch release improves upon the <PREVIOUS_RELEASE_VERSION> release by addressing a few [issues](<MILESTONE_LINK_OF_ISSUES>). -->
+
+You can use the update tool to update to jBallerina <RELEASE_VERSION> as follows.
+
+**For existing users:**
+If you are already using jBallerina version 1.2.14, or above, you can directly update your distribution to jBallerina <RELEASE_VERSION> by executing the following command:
+
+```
+bal dist update
+```
+
+However, if you are using
+
+- jBallerina 1.2.0 to 1.2.13, run `ballerina dist update` to update
+- jBallerina 1.2.0 but being switched to a previous version, run `ballerina dist pull jballerina-<RELEASE_VERSION>` to update
+- a jBallerina version below 1.1.0, install via the [installers](https://ballerina.io/downloads/)
```

Comment:
Since we have included tool version 1.3.1 from 1.2.14 and current patch is 1.2.45. We can assume people have migrated to later versions. Hence we can remove this section and just mention to run `ballerina dist update` to update

Reply:
Removed.